rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/server/RMServiceImpl.java

```diff
@@ -365,53 +372,59 @@ public String getMBeanInfo(String sessionId, String name, List<String> attrs)
             String timeRange) throws RestServerException, ServiceException {
-        return executeFunctionReturnStreamAsString(restClient -> restClient.getNodeMBeansHistory(sessionId,
-                                                                                                 nodeJmxUrl,
-                                                                                                 objectNames,
-                                                                                                 attrs,
-                                                                                                 timeRange));
+        return executeFunctionReturnStreamAsStringWithoutNewLines(restClient -> restClient.getNodeMBeansHistory(sessionId,
+                                                                                                                nodeJmxUrl,
+                                                                                                                objectNames,
+                                                                                                                attrs,
+                                                                                                                timeRange));
     }
 
     @Override
     public String getStatHistory(String sessionId, String range) throws RestServerException, ServiceException {
-        return executeFunctionReturnStreamAsString(restClient -> restClient.getStatHistory(sessionId, range));
+        return executeFunctionReturnStreamAsStringWithoutNewLines(restClient -> restClient.getStatHistory(sessionId,
+                                                                                                          range));
     }
 
     @Override
     public String executeNodeScript(String sessionId, String script, String engine, String nodeUrl)
             throws RestServerException, ServiceException {
-        return executeFunctionReturnStreamAsString(restClient -> restClient.executeNodeScript(sessionId,
-                                                                                              nodeUrl,
-                                                                                              script,
-                                                                                              engine));
+        return executeFunctionReturnStreamAsStringWithoutNewLines(restClient -> restClient.executeNodeScript(sessionId,
+                                                                                                             nodeUrl,
+                                                                                                             script,
+                                                                                                             engine));
+    }
+
+    @Override
+    public String getNodeThreadDump(String sessionId, String nodeUrl) throws ServiceException, RestServerException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)